### DISCUSSION TITLE
USB-C docking station compatible with MonitorControl ?

### DISCUSSION THREAD

**Discussion by mmmago** [MonitorControl/MonitorControl#1456] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

Hello,

I've been using for a while [this USB-C hub ](https://www.amazon.fr/NOVOO-Aluminium-ChromeBook-Matebook-Tablette/dp/B075FGQ988/ref=sr_1_5?__mk_fr_FR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=FUFXS1GZ1CH0&keywords=novoo+usb+c&qid=1697625678&sprefix=novoo+usb+c%2Caps%2C93&sr=8-5) that seems to support DDC with my mac. It has failed me for a while now with intempestive disconnections.

I'm looking for a reliable USB-C / Thunderbolt docking station similar to the [OWC ones](https://www.amazon.fr/OWC-14-Porte-Adaptateur-Voyage-Centimeters/dp/B07N2X64ML), that is compatible with DDC. I don't want to spend that much money on a dock that doesn't support such an essential feature,

Any insights ? Thanks.

**Comment by waydabber** [comment]
I'll move this over to Discussions as it does not seem to be a specific MonitorControl issue but more an inquiry for information from the community. :)

**Reply by mmmago** [reply]
Thanks.

**Comment by bverkron** [comment]
I had an issue that I thought was due to a USB-C dock on my new M1 MacBook Pro, however the issue still happens when I plug directly into the full sized HDMI port on the laptop. The monitor brightness seems to be getting changed via software instead of DDC despite the DDC setting being on and this working on my previous Intel based MacBook Pro 2014.

![image](https://github.com/MonitorControl/MonitorControl/assets/19174503/3e506898-9a3b-4732-983f-30582a904aba)

**Reply by waydabber** [reply]
MonitorControl does not support the HDMI port of any M1 Macs. Try [BetterDisplay](https://betterdisplay.pro) instead as it has hardware DDC support for all connection types.
